nannou_isf/src/pipeline.rs

```diff
@@ -416,7 +416,7 @@ pub fn compile_isf_shader(
         .and_then(|(old_str, isf)| {
             let isf_str = crate::glsl_string_from_isf(&isf);
             let new_str = crate::prefix_isf_glsl_str(&isf_str, old_str);
-            let ty = hotglsl::ShaderType::Fragment;
+            let ty = hotglsl::ShaderStage::Fragment;
```

Comment:
@mitchmindtree Kinda confused? Saw you made some changes last night. This is branched off master where it looks like this built clean.

Reply:
oh, ic it's from #950, nvm!

Reply:
Ahhh I think because `nannou_isf` and the `examples` point at the git branch of `hotglsl` rather than a fixed version, and I updated `hotglsl` last night :sweat_smile: I'll fix these deps in #950 when I rebase it onto this :+1: